build/Common.props

```diff
@@ -62,6 +62,11 @@
     </PackageReference>-->
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
could you move this to non.prod.props so only tests get it.
https://github.com/open-telemetry/opentelemetry-dotnet/pull/2057/files#diff-5e21151922a3cc1af4e89f73a993199997928b4299905fcedc5a9de10d9294d5R18